pyocd/target/builtin/target_HC32F160.py

```diff
@@ -0,0 +1,179 @@
+# pyOCD debugger
+# Copyright (c) 2019 Arm Limited
```

Comment:
Please change the copyright to your own (I guess HDSC). Probably should do that for the other `target_HC32*.py` files too since they are all yours.

Reply:
OK. Thanks.

Reply:
OK. Thanks.